salesagility/SuiteCRM
# Feature codection module issue

Task: SCRM-661

This change includes #4058, #4052,  #4044 and #4049  - Please review and merge this in first.

## Description



This change adds acceptance tests for the issue sugar object. 

## Motivation and Context

Tests issue module:
- create issue module in module builder
- view issue module
- create record with issue module
- filter and select record
- edit record
- duplicate record
- delete record

## How To Test This

* See [tests/README.md](https://github.com/daniel-samson/SuiteCRM/blob/feature_codeception_basic_module/tests/README.md) and configure your environment appropriately
* Run `composer install`
* Run `codecept run install`
* Run `codecept run tests/acceptance/Core/IssueModuleCest.php`

## Types of changes


### Final checklist
